system-tests/projects/create-react-app-custom-index-html/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think we should remove all the nested `.gitignore` files in the community templates, they don't add much value.